uploadcare-rails.gemspec

```diff
@@ -1,29 +1,42 @@
-require_relative 'lib/uploadcare/rails/version'
 
-Gem::Specification.new do |spec|
-  spec.name          = "uploadcare-rails"
-  spec.version       = Uploadcare::Rails::VERSION
-  spec.authors       = ["dmitrij"]
-  spec.email         = ["dmitrij.ivanchenko@gmail.com"]
+$:.push File.expand_path('../lib', __FILE__)
 
-  spec.summary       = %q{TODO: Write a short summary, because RubyGems requires one.}
-  spec.description   = %q{TODO: Write a longer description or delete this line.}
-  spec.homepage      = "TODO: Put your gem's website or public repo URL here."
-  spec.license       = "MIT"
-  spec.required_ruby_version = Gem::Requirement.new(">= 2.3.0")
+# Maintain your gem's version:
+require 'uploadcare/rails/version'
 
-  spec.metadata["allowed_push_host"] = "TODO: Set to 'http://mygemserver.com'"
+# Describe your gem and declare its dependencies:
+Gem::Specification.new do |gem|
+  gem.name          = 'uploadcare-rails'
+  gem.authors       = ["@dmitrijivanchenko (Dmitrij Ivanhcneko)"]
+  gem.email         = ['hello@uploadcare.com']
+  gem.summary       = 'Rails gem for Uploadcare'
+  gem.description   = <<-EOF
+                        Rails wrapper for Uploadcare service.
+                        Based on uploadcare-ruby gem (https://github.com/uploadcare/uploadcare-ruby)
+                      EOF
+  gem.metadata       =  {
+                          'github' => 'https://github.com/uploadcare/uploadcare-rails',
+                          'issue_tracker' => 'https://github.com/uploadcare/uploadcare-rails/issues'
+                        }
+  gem.homepage      = 'https://uploadcare.com/docs/libs/'
+  gem.license       = 'MIT'
 
-  spec.metadata["homepage_uri"] = spec.homepage
-  spec.metadata["source_code_uri"] = "TODO: Put your gem's public repo URL here."
-  spec.metadata["changelog_uri"] = "TODO: Put your gem's CHANGELOG.md URL here."
+  gem.files = Dir['{app,config,db,lib}/**/*', 'MIT-LICENSE', 'Rakefile', 'README.rdoc']
+  gem.test_files = Dir['spec/**/*']
+  gem.version       = Uploadcare::Rails::VERSION
+  gem.add_dependency 'rails', '>= 4'
+  gem.add_dependency 'uploadcare-ruby'
 
-  # Specify which files should be added to the gem when it is released.
-  # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
-  spec.files         = Dir.chdir(File.expand_path('..', __FILE__)) do
-    `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  end
-  spec.bindir        = "exe"
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
-  spec.require_paths = ["lib"]
+  gem.add_development_dependency 'bundler', '~> 1.6'
+  gem.add_development_dependency 'rake'
+  gem.add_development_dependency 'sqlite3'
```

Comment:
Why do we need this?

Reply:
most likely for running tests against example/test projects.